### DISCUSSION TITLE
How to use other MONAI Core neural network architectures in sample apps

### DISCUSSION THREAD

**Discussion by hw-ju** [Project-MONAI/MONAILabel#1516] [Q&A]
Hi there!

Can we use other MONAI Core neural network architectures (e.g., SwinUNETR) in sample apps (e.g., the one for radiology sample app) so that we can train a model from scratch? From `radiology/lib/configs/deepedit.py`, it seems only DynUnet and UNETR are supported. Thanks!

@SanghoonKang503 for visibility.

**Comment by diazandr3s** [comment]
Hi @hw-ju,

Yes, no blockers for this. Just import the network architecture like here https://github.com/Project-MONAI/MONAILabel/blob/main/sample-apps/radiology/lib/configs/segmentation.py#L18, specify the arguments like here: https://github.com/Project-MONAI/MONAILabel/blob/main/sample-apps/radiology/lib/configs/segmentation.py#L82-L90 and train!

**Reply by hw-ju** [reply]
Hi @diazandr3s,
Thank you so much! Very helpful!
Just want to confirm, **any** MONAI Core neural network architecture can be used in the **radiology sample app**, correct?

**Reply by diazandr3s** [reply]
That's correct - any MONAI Core neural network.

**Comment by tangy5** [comment]
Yes, you could also refer to this bundle, but define your own training receipt and train from scratch with SwinUENTR:

https://github.com/Project-MONAI/tutorials/blob/main/monailabel/monailabel_monaibundle_3dslicer_multiorgan_seg.ipynb

After downloading the bundle, you could customize the JSON files, set your own data configs, parameters, and trigger training with or without MONAI Label.

**Comment by hw-ju** [comment]
@diazandr3s @tangy5 Thank you so much!!
